ui/src/store/questions.ts

```diff
@@ -268,6 +268,46 @@ const questionsStore: Module<QuestionsState, RootState> = {
       return Api.Answers(state.question._id)
         .Edit(payload.body, payload.answerId)
         .then(response => commit('replaceAnswer', response.data.data))
+    },
+    async editQuestion({ state, commit }, payload) {
+      if (!state.question) return
+
+      return Api.Questions.Edit(state.question._id, payload).then(response => {
+        commit('storeQuestion', response.data.data)
+      })
+    },
+    async deleteComment({ state, commit }, commentId) {
+      if (!state.question) return
+
+      return Api.Comments(state.question._id)
+        .Delete(commentId)
+        .then(response => {
+          commit('replaceComment', response.data.data)
+        })
+    },
+    async restoreComment({ state, commit }, commentId) {
+      if (!state.question) return
+
+      return Api.Comments(state.question._id)
+        .Restore(commentId)
+        .then(response => {
+          commit('replaceComment', response.data.data)
+        })
+    },
+    async editComment(
+      { state, commit },
+      payload: {
+        commentId: string
+        body: string
+      }
+    ) {
+      if (!state.question) return
```

Comment:
This will work only for comments to questions but not for comments to answers. See how it's managed in `@/api/comments.ts`